netman/adapters/switches/cached.py

```diff
@@ -291,6 +291,11 @@ def set_interface_state(self, interface_id, state):
         self.real_switch.set_interface_state(interface_id, state)
         self.interfaces_cache[interface_id].shutdown = (state == OFF)
 
+    def unset_interface_state(self, interface_id):
+        self.real_switch.unset_interface_state(interface_id)
+        interface = self.real_switch.get_interface(interface_id)
```

Comment:
You don't want to get the interface on real_switch to refresh your cache,
imagine unset is the only thing i do, why refresh the cache?

This call should invalidate the cache for this interface which will be refreshed upon request, not automatically
upon request = get_interface / get_interfaces
just add it to refresh items no?